backend/drive/drive.go

```diff
@@ -466,9 +462,18 @@ func createOAuthClient(name string) (*http.Client, error) {
 	var oAuthClient *http.Client
 	var err error
 
-	serviceAccountPath := config.FileGet(name, "service_account_file")
-	if serviceAccountPath != "" {
-		oAuthClient, err = getServiceAccountClient(serviceAccountPath)
+	// try loading service account credentials from env variable, then from a file
+	serviceAccountCreds := []byte(config.FileGet(name, "service_account_credentials"))
+	if len(serviceAccountCreds) == 0 {
+		serviceAccountPath := config.FileGet(name, "service_account_file")
+		loadedCreds, err := ioutil.ReadFile(os.ExpandEnv(serviceAccountPath))
+		if err != nil {
+			log.Fatalf("Error opening service account credentials file: %v", err)
```

Comment:
Can you make that an error return please?  I've been trying to rid the code of `log.Fatalf` :-)

```go
return nil, errors.Wrap(err, "error opening credentials file")
```